api/spec/services/expense_sheet_calculators/used_days_calculator_spec.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ExpenseSheetCalculators::UsedDaysCalculator, type: :service do
+  let(:calculator) { ExpenseSheetCalculators::UsedDaysCalculator.new(service) }
+  let(:service_range) { get_service_range months: 3 }
+  let(:beginning) { service_range.begin }
+  let(:ending) { service_range.end }
+  let(:service) { create :service, beginning: beginning, ending: ending }
+  let(:created_expense_sheets) { ExpenseSheetGenerator.new(service).create_expense_sheets }
+
+  describe '#used_sick_days' do
+    subject { calculator.used_sick_days }
+
+    context 'with no relevant_for_calculations expense_sheets' do
+      before { created_expense_sheets.each { |expense_sheet| expense_sheet.update sick_days: 10 } }
```

Comment:
Should work as well:
```suggestion
      before { created_expense_sheets.update sick_days: 10 }
```